util/getUniqueRoles.test.js

```diff
@@ -0,0 +1,30 @@
+import getUniqueRoles from './getUniqueRoles';
+
+describe('getUniqueRoles', () => {
+  const MOCK_TEAM_MEMBERS_DATA = {
+    '0001': {
+      jobTitle: 'Developer',
+    },
+    '0002': {
+      jobTitle: 'Developer',
+    },
+    '0003': {
+      jobTitle: 'Developer',
+    },
+    '0004': {
+      jobTitle: 'UI Designer',
+    },
+    '0005': {
+      jobTitle: 'Technical Director',
+    },
+    '0006': {
+      jobTitle: 'Project Manager',
+    },
+  };
+
+  it('returns unique roles from all team members as an array', () => {
+    const roles = getUniqueRoles(MOCK_TEAM_MEMBERS_DATA);
+
+    expect(roles).toEqual(['Developer', 'UI Designer', 'Technical Director', 'Project Manager']);
+  });
+});
```

Comment:
Same goes for here. Let's add some tests for edge cases. Our data isn't always going to place nice with use so that make sure we know this code can handle.